src/tests/ftest/util/server_utils.py

```diff
@@ -230,6 +230,20 @@ def clean_files(self, verbose=True):
             verbose (bool, optional): display clean commands. Defaults to True.
         """
         clean_commands = []
+
+        if self.manager.job.yaml.control_metadata.path:
+            path = self.manager.job.yaml.control_metadata.path
+            # Remove the control metadata directory
+            self.log.info("Cleaning up the %s directory.", str(path))
+            clean_commands.append("sudo rm -fr {}".format(path))
+
+        if self.manager.job.yaml.control_metadata.device:
+            device = self.manager.job.yaml.control_metadata.device
+            # Clean up the control metadata device
+            self.log.info("Cleaning up the %s device.", str(device))
+            clean_commands.append("sudo umount {}".format(device))
+            clean_commands.append("sudo wipefs -a {}".format(device))
```

Comment:
You'll probably want to reverse the order of these, since the path is the mount point for the device.

Reply:
Oh, I typoed the first one. I was thinking that we should `rm -fr $path/*` to nuke the contents but leave the directory itself alone.

Reply:
Thought about it some more and decided that you're right. Best to unmount/wipe first if there's a device, then just nuke the directory. I was thinking that we might want to preserve the directory but nuke the contents, but I think ultimately we'll want to ensure that the directory gets recreated by the control plane.